File: solutions/Javascript/Invert Binary Tree.js
class TreeNode {
  constructor(val, left, right) {
    this.val = val === undefined ? 0 : val;
    this.left = left === undefined ? null : left;
    this.right = right === undefined ? null : right;
  }
}

function invert(node) {
  if (node === null) return;
  let left = node.left;
  let right = node.right;
  node.left = right;
  node.right = left;
  invert(right);
  invert(left);
}
var invertTree = function (root) {
  invert(root);
  return root;
};

File: solutions/python/Invert Binary Tree.py
class TreeNode:
    def __init__(self, val=0, left=None, right=None):
        self.val = val
        self.left = left
        self.right = right
class Solution:
    def invertTree(self, root: Optional[TreeNode]) -> Optional[TreeNode]:
        def inverse(root):
            if root==None: return
            left=root.left;
            root.left=root.right;
            root.right=left;
            inverse(root.left);
            inverse(root.right);
        inverse(root)
        return root